packages/model/src/logic/Scope.ts

```diff
@@ -343,13 +344,14 @@ function injectGlobalAttributes(scope: Model, members: Model[]) {
  */
 function filterWithConformance(
     parent: Model,
+    tags: Set<ElementTag>,
```

Comment:
Since this implies the function is aware of multiple tags, it seems like `selectedMembers` below should also be converted to Record<string, Record<string, Model>> so items with the same name but different tags don't conflict